p2p.go

```diff
@@ -88,7 +88,7 @@ var (
 		Gossip:                   false,
 		ConnectTimeout:           time.Minute,
 		MasterKey:                "",
-		Relay:                    "disable",
+		Relay:                    "nat",
```

Comment:
how's the test result? does it help connect to private IP?

Reply:
need to be tested